frontend/src/forms/CustomerForm.jsx

```diff
@@ -59,6 +59,10 @@ export default function CustomerForm({ isUpdateForm = false }) {
             required: true,
             message: 'Please input your phone!',
           },
+          {
+            pattern: /^(?:[+\d()\-\s]+)$/,
+            message: 'Please enter valid phone number ',
```

Comment:
Other rules end with `!` we probably want to be consistent, and at a minimum remove the empty space as the end.